.rubocop.yml

```diff
@@ -48,3 +48,6 @@ Layout/DotPosition:
 # this is quite aggressive.
 Gemspec/RequiredRubyVersion:
   Enabled: false
+
+Rails/Blank:
```

Comment:
are we sure we want this? This is a rule from rails itself so they do not recommend the usage of `blank?` so i think we might want to do the changes they ask.

Reply:
This repo is not a rails app, but ruby gem. And `blank?` seems to be a rails only thing!

Reply:
uhm 🤔 but this repo failed this check right? so we are using `black?` no?

Reply:
```
Inspecting 30 files
........C.C...................

Offenses:

lib/ibandit/iban.rb:163:10: C: [Correctable] Rails/Blank: Use swift_bank_code.blank? instead of swift_bank_code.nil? || swift_bank_code.empty?.
      if swift_bank_code.nil? || swift_bank_code.empty?
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/ibandit/iban.rb:182:13: C: [Correctable] Rails/Blank: Use swift_branch_code.blank? instead of swift_branch_code.nil? || swift_branch_code.empty?.
      elsif swift_branch_code.nil? || swift_branch_code.empty?
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/ibandit/iban_splitter.rb:20:17: C: [Correctable] Rails/Blank: Use iban.blank? instead of iban.nil? || iban.empty?.
      return if iban.nil? || iban.empty?
                ^^^^^^^^^^^^^^^^^^^^^^^^

30 files inspected, 3 offenses detected, 3 offenses auto-correctable

Exited with code exit status 1

```

Reply:
The cop was complaining about using `empty?` on a string and wants us to use `blank?` (rails thing) instead. So disabling that cop.

Reply:
Hmm interesting, I think we should disable this check for now and I'll fix up gc_ruboconfig

Reply:
yeah sorry! I read the lint error wrong 😅